pkg/controller/cluster/main-controller.go

```diff
@@ -1192,7 +1193,17 @@ func (c *Controller) syncHandler(key string) error {
 				return err
 			}
 
-			if pErr := tenant.CreateConsoleUser(adminClnt, consoleSecret.Data); pErr != nil {
+			skipCreateConsoleUser := false
+			// If Console is deployed with the CONSOLE_LDAP_ENABLED="on" configuration that means MinIO is running with LDAP enabled
+			// and we need to skip the console user creation
+			for _, env := range tenant.GetConsoleEnvVars() {
+				if env.Name == console.ConsoleLDAPEnabled && env.Value == "on" {
+					skipCreateConsoleUser = true
+					break
+				}
+			}
+
+			if pErr := tenant.CreateConsoleUser(adminClnt, consoleSecret.Data, skipCreateConsoleUser); pErr != nil {
```

Comment:
Why not just skip creating console user here vs sending a flag to skip it?

Reply:
We still need to assign the consoleAdmin policy to the console user defined in the Tenant CRD